lib/explorer/data_frame.ex

```diff
@@ -1478,15 +1479,23 @@ defmodule Explorer.DataFrame do
   end
 
   @doc """
-  Creates and modifies columns.
+  Creates or modifies columns based on `Explorer.Query`.
+
+  The query is compiled and runs efficiently against the dataframe.
+  New variables overwrite existing variables of the same name.
+  Column names are coerced from atoms to strings.
+
+  > #### Notice {: .notice}
+  >
+  > Given this function is a macro, you must require `Explorer.DataFrame` before using it.
```

Comment:
```suggestion
  > This is macro, therefore you must `require  Explorer.DataFrame` before using it.
```

Reply:
Another idea is to find all places where we do  `alias Explorer.DataFrame, as: DF` and replace it by `require Explorer.DataFrame, as: DF`.

Reply:
I'm going to try that in the next PR. Thanks! :)